src/common/helpers.js

```diff
@@ -6,7 +6,19 @@
 export function getQueryString(queryObj) {
   return Object.keys(queryObj)
     .filter(key => queryObj[key] !== undefined)
-    .map(key => `${key}=${encodeURI(queryObj[key])}`)
+    .reduce((accum, key) => {
+      // For some query parameters, the key points to an array of values.
+      // In those instances we want a separate param for each value,
+      // otherwise we just want the one param for the value.
+      if (typeof key === 'string') {
```

Comment:
Nice Refactor! but shouldn't we check here for the type of `queryObj[key]` ? And also make sure it works with other types besides `string`.

Reply:
Ariel: The staging server's API is not back yet, so I can't really test the web UI, but I think you are right. Another thing that confuses me is the second and third screenshots. In the second screenshot, when the first organism is selectd, why the URL doesn't have this search parameter? Same thing with the third screenshot when both organisms are selected.

Another issue that I noticed was that `getQueryString` was imported in `src/state/search/actions.js` but was never used. So that import line should be removed.

Reply:
@arielsvn  can you modify the code to address issues that we found out in this PR?

Reply:
@dongbohu thanks for reviewing. I'll address these comments later today.